src/modules/mc_pos_control/mc_pos_control_params.c

```diff
@@ -469,7 +481,31 @@ PARAM_DEFINE_FLOAT(MPC_ACC_UP_MAX, 5.0f);
  * @decimal 2
  * @group Multicopter Position Control
  */
-PARAM_DEFINE_FLOAT(MPC_ACC_DOWN_MAX, 5.0f);
+PARAM_DEFINE_FLOAT(MPC_ACC_DOWN_MAX, 2.0f);
+
+/**
+ * Maximum jerk in manual controlled mode for braking to zero
+ *
+ * @unit m/s/s/s
+ * @min 2.0
+ * @max 15.0
+ * @increment 1
+ * @decimal 2
+ * @group Multicopter Position Control
+ */
+PARAM_DEFINE_FLOAT(MPC_JERK_MAX, 5.0f);
```

Comment:
yep, a guide is needed. I will add info about mc_pos_control parameters in general